source/scripting_v3/GTA/Camera/GameplayCamera.cs

```diff
@@ -444,6 +444,53 @@ public static float FirstPersonAimCameraZoomFactor
 		/// </value>
 		public static bool IsFirstPersonAimCamActive => Function.Call<bool>(Hash.IS_FIRST_PERSON_AIM_CAM_ACTIVE);
 
+		/// <summary>
+		/// Gets a value indicating whether a first person vehicle camera is active.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="GameplayCamera"/> is a first person vehicle camera; otherwise, <see langword="false" />.
+		/// </value>
+		public static bool IsFirstPersonVehicleCamActive => Function.Call<bool>(Hash.IS_CINEMATIC_FIRST_PERSON_VEHICLE_INTERIOR_CAM_RENDERING);
```

Comment:
Wait until you see which bool one `IS_CINEMATIC_FIRST_PERSON_VEHICLE_INTERIOR_CAM_RENDERING` returns when this cinematic cam is rendering, which can be activated by RS button or "Toggle Aircraft Weapon Camera" key (Insert key by default) while in an aircraft with weapons (the image renders the nose of Hydra).
![271590_20230823145954_1](https://github.com/scripthookvdotnet/scripthookvdotnet/assets/9353978/1bdb8f2a-038f-4dce-8f57-f3a4f738f30b)
Same `camCinematicMountedCamera` class, but in this case that native returns false.

Better name suggestion: `IsFirstPersonVehicleInteriorCamRendering`

Reply:
The native returns false because this is not the first-person vehicle cam; it is the so-called "bonnet" cam. I can add a check for it as well, as:

`public static bool IsVehicleNoseCamRendering => Function.Call<bool>(Hash.IS_BONNET_CINEMATIC_CAM_RENDERING)`

Will change the name of the check per your request.

Reply:
This doc from `commands_camera.sch` indicates R* might think the vehicle bonnet cam is also one of the cinematic first-person vehicle cams though. Also, there's "First Person Vehicle Hood" option so the game uses the vehicle hood/bonnet cam for the first-person vehicle view for gameplay cameras instead of the interior cam (I completely forgot that option when I posted the image above).
```
//INFO: 
//PARAM NOTES:
//PURPOSE: Disables the cinematic first-person vehicle bonnet camera for this update only.
NATIVE PROC DISABLE_CINEMATIC_BONNET_CAMERA_THIS_UPDATE() = "0x5ad47d8cfc3a86af"
```

For the vehicle bonnet property, I'd prefer to name it `IsVehicleBonnetCamRendering` considering the "First Person Vehicle Hood" option.